src/theme.rs

```diff
@@ -10,15 +10,17 @@ use crate::{Directory, Error, Result};
 pub(crate) struct Theme {
     path: PathBuf,
     icon_infos: HashMap<String, Vec<IconInfo>>,
-    inherits: Option<String>,
+    inherits: Option<Vec<String>>,
 }
 
 impl Theme {
     pub(crate) fn new<P: AsRef<Path>>(path: P) -> Result<Self> {
         let path = path.as_ref();
         let ini = read_index(path)?;
 
-        let inherits = ini.get::<String>("Icon Theme", "Inherits");
+        let inherits = ini
+            .get::<String>("Icon Theme", "Inherits")
+            .map(|x| x.split(',').map(|x| x.to_string()).collect());
```

Comment:
Not sure it's specified by a standard but I think it's better to have trimmed string here, i.e `x.trim().to_string()`

Reply:
Done.